subsys/bluetooth/host/Kconfig

```diff
@@ -26,7 +26,7 @@ config BT_HCI_CMD_COUNT
 config BT_RX_BUF_COUNT
 	int "Number of HCI RX buffers"
 	default 3 if BT_RECV_IS_RX_THREAD
-	default 20 if (BT_MESH && !BT_DISCARDABLE_BUF_COUNT)
+	default 20 if (BT_MESH && (BT_DISCARDABLE_BUF_COUNT > 0))
```

Comment:
Was there a bug in the logical expression in the first place? I.e. was the logic "inverted" erroneously?

Reply:
fixed that.

Reply:
Actually this should be == 0 since we won't need that many buffer if we can use discardable ones instead.